### ISSUE TITLE
No theming

### ISSUE THREAD

**Issue by Tommatheussen** [kuhnroyal/mat-datetimepicker#9]

**Comment by kuhnroyal** [comment]
I agree that there is room for improvement here, but I am not sure I will find the time to work on this.
My projects always have a custom theme, so there is no immediate need from my side.

For the moment you have to redefine the default theme in your sass file.

**Comment by sapabg** [comment]
I could not make it work by just importing the theme in my styles.css file but I was able to make it work by createing an scss file in my assets folder and including it in my angular.json file in the styles array.
In any other case I got an error processing the scss files.
The file I created is the same as https://github.com/kuhnroyal/mat-datetimepicker/blob/1.x/src/styles.scss except the first line @import '~@angular/material/_theming.scss';

**Comment by Urbanz91** [comment]
@sapabg thanks, it's work for me.
